lib/reporter.js

```diff
@@ -2,18 +2,14 @@
 const chalk = require('chalk');
 
 class Reporter {
-  printReport(report) {
-    report.results.map((commit) => {
+  printReport(commits) {
+    commits.map((commit) => {
```

Comment:
This seemed pretty wack to `map` through something in which you are not even using the return value. Saving to refactor at a later date.

Reply:
you are 100%, it should have been a `forEach` 🙈